spec/unit/matrixrtc/MatrixRTCSession.spec.ts

```diff
@@ -607,91 +427,19 @@ describe("MatrixRTCSession", () => {
             });
         });
 
-        it("does nothing if join called when already joined", () => {
+        it("does nothing if join call when already joined", async () => {
```

Comment:
Not quite sure about changing 'called' to 'call' - I'm not sure how it makes sense as a sentence this way? (Previously it would have been abbreviated from "if the joinRoomSession function is called" but I get the impression the word 'call' is now intended to mean the call as a noun?)